### PR TITLE
Clarify OSH-images folders

### PR DESCRIPTION
**Pull request by evrardjp**

### CODE REVIEW

**Comment by evrardjp** [comment]
DNM: Waiting for #104  first.

File: doc/source/reference.rst [hunk 1]
```diff
@@ -107,14 +107,17 @@ charts.
...
-build OCI compliant images of OpenStack. It follows a source based deployment
-model.
+holds the images used for the OpenStack-Helm project charts. All its images
+are built with Docker.
+
+`openstack-helm-images` provides Dockerfiles directly for all the
+non-OpenStack images.
+
+For the OpenStack images, situated in `openstack/loci/`,
+`openstack-helm-images` only has a shell script, a thin wrapper around `LOCI`.
```
Comment (jsuchome) [inline doc/source/reference.rst]: _only is_ ?

Comment (evrardjp) [reply doc/source/reference.rst]: Maybe I should refactor this whole sentence. Will do tomorrow.

File: doc/source/reference.rst [hunk 2]
```diff
@@ -107,14 +107,20 @@ charts.
...
+non-OpenStack images.
+
+For the OpenStack images, `openstack-helm-images` contains shell scripts,
+situated in `openstack/loci/`. The `build.sh` script is a thin wrapper around
+`LOCI`. `LOCI` is the official OpenStack project to build OCI compliant
+images of OpenStack projects. It uses `docker build` to construct images from
+openstack sources and their requirements are expressed in `bindep` files.
+The `build.sh` runs `LOCI` for the master branch. Other branches can be built
+suing `build-{branchname}.sh` where `branchname` is the name of the OpenStack
+branch (for example, `queens`).
```
Comment (jsuchome) [inline doc/source/reference.rst]: Can you mention the ansible variable that user could change to build for different branch? (`loci_build_command`)

Comment (evrardjp) [reply doc/source/reference.rst]: Will update.

**Pull request merged**